### PR TITLE
fix: update deps, semantic tokens + add more examples

### PR DESCRIPTION
**Pull request by siljec**
Changes:
* Update @warp-ds/uno to 1.0.0 and unocss to 0.55.2
* Update getting-started instructions
* Remove default from color bakground/border/text names (as we did in [this PR in css](https://github.com/warp-ds/css/pull/12/files#diff-6531e41ba379ae34fa944e4bb93b86f21b3ed030afd56db504abdea6c272a7eb))
* Add more examples, based on what we have in [semantic.yml in css/tokens](https://github.com/warp-ds/css/blob/alpha/tokens/finn.no/semantic.yml)
* Add Icongraphy section with Icon Color page
* Use new url to get brand css `https://assets.finn.no/pkg/@warp-ds/css/v1/tokens/<brand>` vs old url `https://assets.finn.no/pkg/@warp-ds/tokens/v1/<brand>`

Other notes:
* We are missing documentation for`s-focused` and `s-divider`. Where does it belong in the CSS docs?

### CODE REVIEW

File: docs/.vitepress/config.js
Comment (siljec) [inline docs/.vitepress/config.js:34]: Remember to replace this with v1
```suggestion
        href: 'https://assets.finn.no/pkg/@warp-ds/css/v1/tokens/finn-no.css',
```

File: docs/classes.list.js
Comment (siljec) [inline docs/classes.list.js:119]: We have removed `s-border-xxx-active-hover` (You can check [here](https://docs.google.com/spreadsheets/d/1Q-Tr_dwJVfxgh3527IFjXQKqtnPPM42QeIr-M6q-zu8/edit#gid=888578831) and it is not in semantic.yml anymore)

File: docs/classes.list.js
Comment (siljec) [inline docs/classes.list.js:585]: We have these classes (You can check [here](https://docs.google.com/spreadsheets/d/1Q-Tr_dwJVfxgh3527IFjXQKqtnPPM42QeIr-M6q-zu8/edit#gid=888578831) and it is not in semantic.yml anymore)

File: docs/classes.list.js
Comment (siljec) [inline docs/classes.list.js:45]: These are not in `semantic.yml` file, not sure if I should keep them or not. Not mentioned in the spreadsheet either.

**Comment by siljec** [comment]
image
image
image
image
image

**Comment by BalbinaK** [comment]
> Use new url to get brand css https://assets.finn.no/pkg/@warp-ds/css/v1/tokens/<brand>

We have more changes that are not available on v1 yet (starting from 1.0.0-alpha.36). Maybe we should continue using specific version numbers until 1.0.0 is released and aliased. Wdyt?

**Comment by siljec** [comment]
> We have more changes that are not available on v1 yet (starting from 1.0.0-alpha.36). Maybe we should continue using specific version numbers until 1.0.0 is released and aliased. Wdyt?

Yes, I wasn't sure if we want to merge this now, or wait until after the major release? If we want to merge this now, I can change to a specific version? @BalbinaK

**Comment by BalbinaK** [comment]
Ah, of course! I forgot that this PR should not be merged until after the 1.0.0 of @warp-ds/css is released!

**Comment by siljec** [comment]
I updated the PR title now, just to be on the safe side 😛 Fyi, I have a checklist of what we need to do after the majore release [here](https://nmp-jira.atlassian.net/browse/WARP-136) @BalbinaK

File: docs/classes.list.js
Comment (BalbinaK) [reply docs/classes.list.js:45]: Hm. They are part of [color rules in warp-ds/drive](https://github.com/warp-ds/drive/blob/alpha/src/_rules/color.js), so I don't think we should remove them. Wdyt @Skadefryd?

File: docs/.vitepress/config.js
Comment (BalbinaK) [inline docs/.vitepress/config.js:34]: I think we also need to add [fonts](https://assets.finn.no/pkg/@warp-ds/fonts/v1/finn-no.css) here now that they are no longer included in the brand CSS. I suppose it would be apparent in the [Font Size page](https://warp-ds.github.io/css-docs/font-size#setting-the-font-size).

File: docs/.vitepress/theme-switcher-listener.js
Comment (BalbinaK) [inline docs/.vitepress/theme-switcher-listener.js:8]: Since the theme switching logic was previously only replacing this one href with brand CSS, and that doesn't include fonts, I believe we could keep the logic but also load all font setups in the docs head (for Tori, Blocket, Finn). Having all fonts defined this way shouldn't cause any trouble because the font that is picked up is the one that is actually used like e.g. body, :host { font-family: "Finntype"; } (based on the brand CSS).

File: docs/classes.list.js
Comment (BalbinaK) [inline docs/classes.list.js:156]: We also added `s-border-negative-selected` and `s-border-negative-selected-hover` in [@warp-ds/css@1.0.0-alpha.36](https://github.com/warp-ds/css/commit/1d3bfdb0787eedcff16782099c5657138f607826#diff-2704f7af6af729d92f7b4b60741f5e6e2c8bfbedf35b98f0aaae6aa1ceca1570)

**Review by BalbinaK — COMMENTED** [review]
Looks great! Just a few comments 💪

**Review by Skadefryd — APPROVED** [review]
Looks great :)

**Comment by BalbinaK** [comment]
@siljec sorry for rebasing instead of merging `master` changes - I thought that would simplify the commit history here but 
I ended up with git overriding commit authors 😅 

Changes since last time were updated in the PR description and included:
- add all font definitions in head to simplify theme-switching logic
- update unocss, unocss/core to 0.55.2
- update @warp-ds/uno to 1.0.0
- update getting-started instructions

File: docs/icon-color.md
Comment (AnnaRybkina) [inline docs/icon-color.md:21]: Minor neat pick, feel free to ignore
```suggestion
This section will be updated when Warp starts to support icons.
```

**Review by AnnaRybkina — APPROVED** [review]
I went through the code and tested locally. I think it looks great and we've added really valuable information to make our docs better. Great work

**Pull request merged**
